addons/spreadsheet/static/src/global_filters/components/filter_date_from_to_value/filter_date_from_to_value.js

```diff
@@ -0,0 +1,38 @@
+/** @odoo-module */
+
+import { Component } from "@odoo/owl";
+import { _t } from "@web/core/l10n/translation";
+import { DateTimeInput } from "@web/core/datetime/datetime_input";
+import { serializeDate, deserializeDate } from "@web/core/l10n/dates";
+
+export class DateFromToValue extends Component {
+    static template = "spreadsheet_edition.DateFromToValue";
```

Comment:
Template prefix should be the module. (it's a mistake we did on a few other templates, that's probably where you copied it)
```suggestion
    static template = "spreadsheet.DateFromToValue";
```